common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -35,4 +36,11 @@ public class RssClientConf {
       .intType()
       .defaultValue(3)
       .withDescription("The zstd compression level, the default level is 3");
+
+  public static final ConfigOption<ShuffleDataDistributionType> DATA_DISTRIBUTION_TYPE = ConfigOptions
+      .key("rss.client.shuffle.data.distribution.type")
```

Comment:
What's the relation with `aqe` config option?

Reply:
The local order is just a type, should not be bound to any spark optimization. Although it will benefit for spark AQE.

Reply:
OK, got it. Should we add some documents for this feature?

Reply:
Of course. It will be attached later

Reply:
Done